app/javascript/src/components/Notes/index.jsx

```diff
@@ -0,0 +1,108 @@
+// @ts-nocheck
+import React, { useState, useEffect } from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import Card from "components/Common/Card";
+import NewPane from "components/Common/CreatePane";
+import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
+
+import { MENUBAR_DATA, NOTES_FORM_INITIAL_FORM_VALUES } from "./constants";
+import { NOTES_DATA } from "./constants";
+import Form from "./Form";
+
+import DeleteAlert from "../Common/DeleteAlert";
+
+const Notes = () => {
+  const [showNewNotePane, setShowNewNotePane] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [notes, setNotes] = useState([]);
+  const [category, setCategory] = useState("All");
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [, setSelectedNoteIds] = useState(null);
```

Comment:
What's the point of declaring a hook setter without a state? Couldn't you use a variable here rather? What are the caveats?

Reply:
When I declared that variable, it was it was giving the error of variable not used is pre commit check as no actual request was made using that Id

Reply:
If selected note ids aren't used then what's the point of this state at all?

Reply:
Yes, as no request is made its of no use. Removed it. Actually added in the start just to understand the functionality.